redhat-upgrade-tool.py

```diff
@@ -365,6 +368,17 @@ def handle_sigwinch(signum, frame):
         upgrade_boot_args()
         prep_boot(kernel, initrd)
 
+    # Check for available space in /boot/ needed for kernel and grub
+    # installation during the upgrade.
+    statvfs = os.statvfs("/boot")
+    avail_bytes = statvfs.f_frsize * statvfs.f_bavail
+    if avail_bytes < MIN_AVAIL_BYTES_FOR_BOOT:
+        reset_boot()
+        additional_mb_needed = (MIN_AVAIL_BYTES_FOR_BOOT - avail_bytes) / 1e6
+        sys.stderr.write("Not enough space. /boot/ needs additional %d MB.\n"
+                         % int(additional_mb_needed))
```

Comment:
1. I'll remove calling int(), I didn't know that %d does that for me.
2. I'll add calling gettext here. Thanks for catching that.
3. When I ran df -h, I've seen M, so I thought it's mega, but it's mibi actually.
    However, when you run dnf and you don't have enough space, it gives you: "Disk Requirements: At least 32MB more space needed on the /boot filesystem."
    These are the two tools I ran to decide what unit to use, so you can see why I chose mega. But you're right, mega may be perceived as deprecated, I'll use mibi.